domovoi.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import shlex
+import sys
+import time
+
+import settings
+
+from solar_car_process import SolarCarProcess
+
+
+class Domovoi:
+    def start_processes(self, solar_car_processes):
+        for solar_car_process in solar_car_processes:
+            try:
+                solar_car_process.start()
+            except OSError as e:
+                logging.critical(e)
+                self.kill_processes(solar_car_processes)
+                raise
+
+    def kill_processes(self, solar_car_processes):
+        for solar_car_process in solar_car_processes:
+            try:
+                solar_car_process.process.kill()
+            except AttributeError:
+                pass
```

Comment:
What is the purpose of this try except? Usually you should not be catching exceptions you don't know how to deal with.